integ-test/src/test/java/org/opensearch/sql/ppl/NewAddedCommandsIT.java

```diff
@@ -119,6 +119,38 @@ public void testAppendcol() throws IOException {
     verifyQuery(result);
   }
 
+  @Test
+  public void testValuesFunction() throws IOException {
```

Comment:
can we add one test to cross cluster search IT?

Reply:
added here https://github.com/opensearch-project/sql/pull/4042/commits/02a81298a8503e95b65d281d7a57403954409fd2